docs/managed-datahub/release-notes/v_0_3_11.md

```diff
@@ -0,0 +1,99 @@
+# v0.3.11
+
+---
+
+## Release Availability Date
+
+12-May-2025
+
+## Recommended CLI/SDK
+
+1.0.0.3
+
+## Helm Chart
+
+1.5.32
+
+## Known Issues
+
+## Release Changelog
+
+---
+
+### v0.3.11
+
+- Breaking Changes
+
+- #13004: The `acryl-datahub-airflow-plugin` dropped support for Airflow 2.3 and 2.4.
+- #13186: NoCode Migration Removed - This code hasn't been required in many years. If needed, a user should upgrade to DataHub 1.0.x prior to upgrading to a later version.
+- #13397: `async_flag` removed from rest emitter, replaced with emit mode ASYNC
```

Comment:
I feel like this may impact some users. Let's keep our eyes and ears out.

Reply:
Remove this text, it was deprecated and will redirect to ASYNC mode.